bio_hansel/qc/checks.py

```diff
@@ -41,16 +41,56 @@ def is_missing_tiles(st: Subtype, df: DataFrame, p: SubtypingParams) -> Tuple[Op
     Returns:
         None, None if less missing tiles than tolerate; otherwise, "FAIL", error message
     """
-    if not st.are_subtypes_consistent:
-        return None, None
+    error_status = None
+    error_messages = None
+
+    if st.are_subtypes_consistent:
+        error_status, error_messages = check_for_missing_tiles(is_fastq=st.is_fastq_input(),
+                                                               curr_subtype=st.subtype,
+                                                               df=df,
+                                                               exp=int(st.n_tiles_matching_all_expected),
+                                                               obs=int(st.n_tiles_matching_all),
+                                                               p=p)
+    else:
+        error_status = []
```

Comment:
`error_status` is initialized as a `list`, but later it's reassigned a string or `None`. Reassigning the same var with a different type can get confusing quickly esp. in a dynamically typed lang like Python. So it's best to avoid it when you can. Same goes for `error_messages`. 

Also your overall QC status will already be a fail from it being inconsistent in this case so you can ignore it from `check_for_missing_tiles` (i.e. `_, msg = check_for_missing_tiles(...)` where `_` means the var is ignored). Also you can drop `error_` from the var names to reduce verbosity.

Reply:
Ah gotcha, that makes a lot of sense. I think I'll make a temporary variable that will hold the messages received from `check_for_missing_tiiles()` and then assign `error_messages` to the joined messages.

Reply:
Also, thanks for the `_` tip, I was wondering how to do that.

Reply:
Yeah a temp var would be nice in this case. You could try to turn it into a really long list comp haha...

Reply:
Do you think we should keep `error_` variable names? 
We have a lot of local variables like `curr_messages`, `coverage_msg`, etc...